lib/ui/views/home/widgets/restaurant_card/restaurant_card.dart

```diff
@@ -17,11 +19,10 @@ class RestaurantCard extends StatelessWidget {
     return ViewModelBuilder<RestaurantCardModel>.reactive(
       onModelReady: (model) => model.calculateDistanceFromUser(),
```

Comment:
the restaurant object would already contain the distance from user as it is calculated on initialisation